quinn-h3/src/proto/frame.rs

```diff
@@ -5,15 +5,20 @@ use quinn_proto::{
     coding::{BufExt, BufMutExt, Codec, UnexpectedEnd},
     VarInt,
 };
-use std::collections::HashSet;
 use tracing::trace;
 
+use super::settings::{Error as SettingsError, SettingId, SettingsFrame};
+
 #[derive(Debug, PartialEq)]
 pub enum Error {
     Malformed,
     UnsupportedFrame(u64),
     UnexpectedEnd,
     InvalidFrameValue,
+    InvalidSettingId(u64),
+    InvalidSettingValue(SettingId, u64),
+    SettingRepeated(SettingId),
+    SettingsExceded,
```

Comment:
Typo: exceeded.

Reply:
Rebased in cfea581d H3: refactor settings in it's own module